hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AzureBlobIngressHandler.java

```diff
@@ -119,21 +115,21 @@ protected AbfsRestOperation remoteWrite(AbfsBlock blockToUpload,
       TracingContext tracingContext)
       throws IOException {
     reqParams.setBlockId(blockToUpload.getBlockId());
-    reqParams.setEtag(getETag());
     AbfsRestOperation op;
+    long threadId = Thread.currentThread().getId();
+    String threadIdStr = String.valueOf(threadId);
     TracingContext tracingContextAppend = new TracingContext(tracingContext);
-    tracingContextAppend.setIngressHandler("BAppend");
+    tracingContextAppend.setIngressHandler("BAppend " + "T" + threadIdStr);
```

Comment:
Since BAppend and T are constant string so we can merge them both as one string.
`tracingContextAppend.setIngressHandler("BAppend T" + threadIdStr);`